dev/dd/datarepo-api.yaml

```diff
@@ -8,7 +8,8 @@ env:
   SPRING_PROFILES_ACTIVE: google,cloudsql,dev,dd,perftest
   DB_STAIRWAY_USERNAME: drmanager
   DB_STAIRWAY_URI: jdbc:postgresql://dd-jade-gcloud-sqlproxy.dd:5432/stairway-dd
-  DB_DATAREPO_URI: jdbc:postgresql://dd-jade-gcloud-sqlproxy.dd:5432/datarepo-dd 
+  DB_DATAREPO_URI: jdbc:postgresql://dd-jade-gcloud-sqlproxy.dd:5432/datarepo-dd
+  DATAREPO_DNSNAME: jade.datarepo-dev.broadinstitute.org
```

Comment:
I think there are separate dns names for each namespace aren't there? I think mine should be
`jade-dd.datarepo-dev.broadinstitute.org`

Reply:
Oh, very interesting. 
So, my thought was that anything in the "[application-dev.properties](https://github.com/DataBiosphere/jade-data-repo/pull/574/files#diff-60a8da0b7574e136eb40838da2f2846dL1)" would have been applied to any namespace in the dev environment. I think that was a bad assumption. I'm going to test this out on my personal namespace. 

If this is the case, then I don't think we have dnsName defined anywhere for our personal namespaces.

Reply:
I did a test run and confirmed that jade-sh.datarepo-dev.broadinstitute.org works for my dev namespace.  So, I made the switch! 

But, if I'm understanding correctly, we didn't have this set correctly for our person dev namespaces before. So, I'm confused as to why we haven't run into issues before.